docs/transpile/transpiler-stages.ipynb

```diff
@@ -125,7 +124,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This distribution is quite wide, signaling the difficulty the SWAP mapper is having in computing the best mapping.\n",
+    "This distribution is quite wide, signaling the difficulty the SWAP mapper is having in computing the best mapping.  In fact, we can take a closer look at what hardware qubits were chosen to run the circuit:\n",
+    "\n",
+    "<img src=\"/images/transpile/transpiler-stages/hardware-qubit-graph.png\" alt=\"Layout of qubits on hardware\" width=\"350%\" />\n",
+    "\n",
+    "As you can see this selection of qubits isn't even fully connected to each other!  This means unnecessary SWAP gates must be inserted in order to execute all of the two-qubit gates because we naively chose which hardware qubits to use in our circuit.\n",
```

Comment:
```suggestion
    "This distribution is quite wide, which demonstrates how difficult it is for the SWAP mapper to compute the best mapping. Look closely at which hardware qubits were chosen to run the circuit, and note that this selection of qubits isn't fully connected to each other.\n",
    "\n",
    "<img src=\"/images/transpile/transpiler-stages/hardware-qubit-graph.png\" alt=\"Layout of qubits on hardware\" width=\"350%\" />\n",
    "\n",
    "This requires inserting unnecessary SWAP gates to execute all the two-qubit gates because we naively chose which hardware qubits to use in our circuit.\n",
```